nana/modules/admins.py

```diff
@@ -153,194 +189,200 @@ async def pin_message(client, message):
-                            permissions=mute_permission,
-                            until_date=int(time.time() + 86400),
-                        )
-                    else:
-                        await client.restrict_chat_member(
-                            chat_id=message.chat.id,
-                            user_id=message.reply_to_message.from_user.id,
-                            permissions=mute_permission,
-                        )
-                    await message.delete()
-                except Exception as e:
-                    await edrep(message, text="`Error!`\n" f"**Log:** `{e}`")
-                    return
-            else:
-                await edrep(message, text=tld('mute_user'))
-                await asyncio.sleep(5)
+            try:
+                if message.reply_to_message:
+                    user_id = message.reply_to_message.from_user.id
+                else:
+                    usr = await client.get_users(message.command[1])
+                    user_id = usr.id
+            except IndexError:
+                await message.reply('some ooga booga')
+                return
+            try:
+                await client.restrict_chat_member(
+                    chat_id=message.chat.id,
+                    user_id=user_id,
+                    permissions=mute_permission
+                )
                 await message.delete()
+            except Exception as e:
+                await edrep(message, text="`Error!`\n"
+                        f"**Log:** `{e}`"
+                    )
+                return
         else:
             await edrep(message, text=tld('denied_permission'))
-            await asyncio.sleep(5)
-            await message.delete()
     else:
         await message.delete()
 
 
 @app.on_message(filters.user(AdminSettings) & filters.command("unmute", Command))
 async def unmute(client, message):
-    if message.chat.type in ["group", "supergroup"]:
+    if message.chat.type in ['group', 'supergroup']:
+        chat_id = message.chat.id
         can_unmute = await admin_check(message)
         if can_unmute:
             try:
                 if message.reply_to_message:
-                    await client.restrict_chat_member(
-                        chat_id=message.chat.id,
-                        user_id=message.reply_to_message.from_user.id,
-                        permissions=unmute_permissions,
-                    )
+                    user_id = message.reply_to_message.from_user.id
                 else:
-                    await edrep(message, text=tld('unmute_user'))
-                    await asyncio.sleep(5)
+                    usr = await client.get_users(message.command[1])
+                    user_id = usr.id
+            except IndexError:
+                await edrep(message, text='must give a user to unmute')
+                return
+            try:
+                get_mem = await client.get_chat_member(
```

Comment:
Codacy found an issue: [local variable 'get_mem' is assigned to but never used (F841)](https://app.codacy.com/gh/MadeByThePinsHub/Nana-Remix/pullRequest?prid=6330176)